api/handlers.py

```diff
@@ -29,6 +29,8 @@ class User(Base):
     phone_number = Column('phone_number', BigInteger)
     contact = Column('contact', Boolean, nullable=False)
     active = Column('active', Boolean, nullable=False)
+    oauth_provider = Column('oauth_provider', String)
+    oauth_id = Column('oauth_id', String)
```

Comment:
I feel like this may not be needed once we use flask-login and its relevant OAuth stuff. We can always add these(if needed) when we figure out OAuth.